modules/i3s/src/helpers/i3s-nodepages-tiles.js

```diff
@@ -92,8 +84,50 @@ export default class I3SNodePagesTiles {
       attributeUrls,
       materialDefinition,
       textureFormat,
-      children
+      children,
+      isDracoGeometry
+    });
+  }
+
+  _getContentUrl(meshGeometryData) {
+    let result = {};
+    const geometryDefinition = this.tileset.geometryDefinitions[meshGeometryData.definition];
+    let geometryIndex = -1;
+    if (this.options.i3s && this.options.i3s.useDracoGeometry) {
+      geometryIndex = geometryDefinition.geometryBuffers.findIndex(
+        buffer => buffer.compressedAttributes && buffer.compressedAttributes.encoding === 'draco'
+      );
+    }
+    if (geometryIndex === -1) {
+      geometryIndex = geometryDefinition.geometryBuffers.findIndex(
+        buffer => !buffer.compressedAttributes
+      );
+    }
```

Comment:
Use `else` ?

Reply:
No. After this `if` `geometryIndex` can get >=0

Reply:
I see. A few short comments can help guide the reader.